workflows-scripts/item-search-filter/jmeter-supported-data/credentials.csv

```diff
@@ -0,0 +1 @@
+folio,folio,fs09000000

```

Comment:
How about replacing real credentials to the template [folio username],[foolio password],[tenant Id]?